modules/wechat_qrcode/test/test_qrcode.cpp

```diff
@@ -283,6 +284,52 @@ TEST_P(Objdetect_QRCode_Multi, regression) {
     }
 }
 
+TEST(Objdetect_QRCode_points_position, rotate45) {
+    string path_detect_prototxt, path_detect_caffemodel, path_sr_prototxt, path_sr_caffemodel;
+    path_detect_prototxt = findDataFile("../dnn/wechat/detect.prototxt", false);
+    path_detect_caffemodel = findDataFile("../dnn/wechat/detect.caffemodel", false);
+    path_sr_prototxt = findDataFile("../dnn/wechat/sr.prototxt", false);
+    path_sr_caffemodel = findDataFile("../dnn/wechat/sr.caffemodel", false);
```

Comment:
> `../dnn/`

Invalid path prefix.

DNN models are stored in location specified by `OPENCV_DNN_TEST_DATA_PATH`.
Don't mess them with `opencv_extra` location (and don't run "download" scripts there).

You need to use this approach: https://github.com/opencv/opencv/blame/4.6.0/modules/objdetect/test/test_main.cpp#L10-L25

Reply:
fixed